srtcore/api.cpp

```diff
@@ -3674,7 +3666,7 @@ int CUDT::epoll_release(const int eid)
 
 CUDTException& CUDT::getlasterror()
 {
-   return *s_UDTUnited.getError();
+   return s_UDTUnited.getError();
```

Comment:
This forwarder is completely needless now. You can call `GetThreadLocalError` directly here.